tm2/pkg/bft/rpc/config/config.go

```diff
@@ -88,7 +88,7 @@ type RPCConfig struct {
 // DefaultRPCConfig returns a default configuration for the RPC server
 func DefaultRPCConfig() *RPCConfig {
 	return &RPCConfig{
-		ListenAddress:          "tcp://127.0.0.1:26657",
+		ListenAddress:          "tcp://0.0.0.0:26657",
```

Comment:
i'm 100% Again this. This is WAY TO DANGEROUS for mainnet.

Too many validators on cosmos with having `127.0.0.1` by default expose their validator on `0.0.0.0`, which is a terrible error and it's dangerous.

The result will be way worse if we set it to `0.0.0.0` by default.